test/end_to_end/conftest.py

```diff
@@ -1,15 +1,17 @@
 import os
 import numpy as np
 import pytest
+import sys
 
 import list_of_cases
 
 
 @pytest.fixture
-def test_data():
+def test_data(capsys):
```

Comment:
You can remove the capsys. Sorry this is actually something I added lol. Not sure why it shows up here. Anyways it's not needed anymore